tgui/packages/tgui/interfaces/Cargo.js

```diff
@@ -485,58 +483,89 @@ const Category = (props, context) => {
 
   const {
     selectedPackCat,
+    should_filter,
     level,
   } = props;
 
+  const [
+    filter,
+    setFilter,
+  ] = useLocalState(context, `pack-name-filter`, null);
+
+  const filterSearch = (entry) =>
+    should_filter
+      ? !filter // If we don't have a filter, don't filter
+        ? true // Show everything
+        : supplypackscontents[entry].name
+          ?.toLowerCase()
+          .includes(filter.toLowerCase()) // simple contains search
+      : true;
```

Comment:
```suggestion
const filterSearch = entry =>
  should_filter && filter
    ? supplypackscontents[entry].name
        ?.toLowerCase()
        .includes(filter.toLowerCase())
    : true;
```

Any reason you couldn't simplify this?